subsys/shell/Kconfig

```diff
@@ -146,7 +146,7 @@ config SHELL_CMDS_SELECT
 	  command. Exit to main command tree is with alt+r.
 
 config SHELL_LOG_BACKEND
-	bool
+	bool "Enable shell log backend"
 	depends on !LOG_MINIMAL
 	default y if LOG
```

Comment:
please add help information which will state that this will disable support for logging on all instances of shell.